src/main/java/de/themorpheus/edu/taskservice/controller/solution/MultipleChoiceSolutionController.java

```diff
@@ -83,4 +72,16 @@ public void deleteAll(int taskId) {
 		return ControllerResult.of(dto);
 	}
 
+	@Override
+	public void deleteAll(int taskId) {
+		ControllerResult<SolutionModel> optionalSolution = this.solutionController.getOrCreateSolution(taskId, NAME_KEY);
+		if (optionalSolution.isResultNotPresent()) return;
+
+		List<MultipleChoiceSolutionModel> multipleChoiceSolutionModels = this.multipleChoiceSolutionRepository.findAllById(
```

Comment:
1. Why do we need the collection as a parameter instead of a single integer?
2. Use "count" instead of transferring ALL solutions over the network

Reply:
Using "exist" would be perfect, please evaluate this option

Reply:
What do you mean with count

Reply:
You just want to know if there are entries matching your query, for this u could use the count of matching entries